pkg/authorization/api/register.go

```diff
@@ -56,30 +56,3 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&ClusterRoleList{},
 	)
 }
-
-func (obj *ClusterRoleList) GetObjectKind() unversioned.ObjectKind          { return &obj.TypeMeta }
-func (obj *ClusterRoleBindingList) GetObjectKind() unversioned.ObjectKind   { return &obj.TypeMeta }
-func (obj *ClusterPolicyBindingList) GetObjectKind() unversioned.ObjectKind { return &obj.TypeMeta }
-func (obj *ClusterPolicyList) GetObjectKind() unversioned.ObjectKind        { return &obj.TypeMeta }
-func (obj *ClusterPolicyBinding) GetObjectKind() unversioned.ObjectKind     { return &obj.TypeMeta }
-func (obj *ClusterPolicy) GetObjectKind() unversioned.ObjectKind            { return &obj.TypeMeta }
```

Comment:
> because TypeMeta implements it, so anything that inlines TypeMeta gets it

So I should be seeing a commit that removes this declaration from every API type, right?